patches/minecraft/net/minecraft/world/level/block/entity/HangingSignBlockEntity.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/level/block/entity/HangingSignBlockEntity.java
++++ b/net/minecraft/world/level/block/entity/HangingSignBlockEntity.java
+@@ -10,6 +_,7 @@
+    public HangingSignBlockEntity(BlockPos p_250603_, BlockState p_251674_) {
+       super(BlockEntityType.f_244529_, p_250603_, p_251674_);
+    }
++   public HangingSignBlockEntity(BlockEntityType type, BlockPos pos, BlockState state) {super(type, pos, state);}
```

Comment:
The new constructor is added as a single line, in accordance with [the patch guidelines](https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge#conventions-for-coding-patches-for-a-minecraft-class-javapatch) instruction to minimize lines of code changed.

Reply:
Typically the rule is 'follow the style around you'

This should be 4 lines:
```diff
+    
+    public HangingSignBlockEntity(BlockEntityType type, BlockPos pos, BlockState state) {
+        super(type, pos, state);
+    }
```
As it fits the style around it. The point is to make the patches small and maintainable. not to make the code unreadable.
Basically don't indent existing code, don't add 100's of lines in patches etc.

Reply:
Thank you for letting me know; I'll try to update this tomorrow.

Reply:
I have updated the patch as you recommended, please let me know if there is anything else you would like changed.